frontend/src/assets/icons/SignatureIcon.tsx

```diff
@@ -0,0 +1,26 @@
+// hands-clapping icon taken from  https://phosphoricons.com/.
+
+export const SignatureIcon = (): JSX.Element => {
+  return (
+    <svg
+      width="24"
+      height="24"
+      viewBox="0 0 24 24"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
```

Comment:
Generally we try to add `props` so that it can be altered by the parent component. i.e., `<SignatureIcon width="1rem" height="1rem" />` if required.

```suggestion
export const SignatureIcon = (props: React.SVGProps<SVGSVGElement>): JSX.Element => {
  return (
    <svg
      width="24"
      height="24"
      viewBox="0 0 24 24"
      fill="none"
      xmlns="http://www.w3.org/2000/svg"
      {...props}
```

Reply:
Thanks for the tip! Makes complete sense.